Dockerfile

```diff
@@ -13,6 +13,6 @@ COPY . /usr/src/app/
 
 RUN yarn install
 
-EXPOSE 3000
+EXPOSE 443 80
```

Comment:
Ah, I just realized we may want to expose `3000` in addition to the two others since in the `www` file we use `3000` as a fallback if `PORT` isn't set..